x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -345,15 +357,33 @@ export class AlertsClient {
         },
         operation
       );
-      return buildEsQuery(
+      let esQuery;
+      if (id != null) {
+        esQuery = { query: `_id:${id}`, language: 'kuery' };
```

Comment:
It could be helpful to have a comment here describing how these different scenarios happen